third_party/move/move-model/src/ast.rs

```diff
@@ -261,28 +261,31 @@ impl Condition {
 pub type PropertyBag = BTreeMap<Symbol, PropertyValue>;
 
 /// The value of a property.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub enum PropertyValue {
     Value(Value),
     Symbol(Symbol),
     QualifiedSymbol(QualifiedSymbol),
 }
 
 /// Specification and properties associated with a language item.
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Clone, Default, PartialEq, Eq, Hash)]
```

Comment:
I assume you're adding Hash here because we need it somewhere.  Where is the HashMap/Set that needs this to be hashed?

Reply:
It's because this data now became part of `ExpData` which in turn is internalized. So the `internement` crate needs this.